modules/oe_whitelabel_list_pages/tests/src/Functional/ListPagesTest.php

```diff
@@ -121,13 +121,8 @@ protected function createListPage(): NodeInterface {
    *   Container with the expected elements.
    */
   protected function assertFacetsSummaryTitle(int $expected_count, ElementInterface $container): void {
-    $title = $container->find('css', 'div.col-md-6.col-lg-8 h4');
-    $text = $title->find('css', 'span.text-capitalize');
-
-    $this->assertSame(
-      sprintf('%s (%s)', $text->getText(), $expected_count),
-      $title->getText()
-    );
+    $title = $container->find('css', 'div.col-md-6.col-lg-8 h2.text-capitalize');
+    $this->assertSame('News list page ' . sprintf('(%s)', $expected_count), $title->getText());
```

Comment:
I like this simplification. The list page title is always the same, so no reason to make it dynamic.

Although, the usage of sprintf is a bit strange here. Either use string concat all the way or use sprintf() for the complete string.
```suggestion
    $this->assertSame(sprintf('News list page (%s)', $expected_count), $title->getText());
```
or
```suggestion
    $this->assertSame('News list page (' . $expected_count . ')'), $title->getText());
```